spec/tools/environment_builders/openstack/helper_methods.rb

```diff
@@ -27,7 +27,7 @@ def create_or_update_ems(name, hostname, password, port, userid, version)
                     :ipaddress             => hostname,
                     :port                  => port,
                     :api_version           => version,
-                    :zone                  => Zone.first,
+                    :zone                  => Zone.visible.first,
```

Comment:
I believe this was also resolved by Adam's PR to fix the Zone.first in specs.

Reply:
this file was not included

Reply:
Interesting, do tests fail without this change?  If so, we need to fix that code/test.  Thanks!

Reply:
This isn't called as part of the spec tests, this is to build up an env to record VCRs, this should probably be Zone.default_zone